src/Fixer/Comment/HashToSlashCommentFixer.php

```diff
@@ -12,48 +12,95 @@
 
 namespace PhpCsFixer\Fixer\Comment;
 
-use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\Fixer\WhitespacesAwareFixerInterface;
 use PhpCsFixer\FixerDefinition\CodeSample;
 use PhpCsFixer\FixerDefinition\FixerDefinition;
-use PhpCsFixer\Tokenizer\Token;
 use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Utils;
+use PhpCsFixer\WhitespacesFixerConfig;
 
 /**
  * Changes single comments prefixes '#' with '//'.
  *
  * @author SpacePossum
+ *
+ * @deprecated in 2.4, proxy to SingleLineCommentStyleFixer
  */
-final class HashToSlashCommentFixer extends AbstractFixer
+final class HashToSlashCommentFixer implements DefinedFixerInterface, WhitespacesAwareFixerInterface
 {
+    private $realFixer;
+
+    public function __construct()
+    {
+        $this->realFixer = new SingleLineCommentStyleFixer();
+        $this->realFixer->configure(['comment_type' => 'hash']);
+    }
+
     /**
      * {@inheritdoc}
      */
-    public function getDefinition()
+    public function isCandidate(Tokens $tokens)
     {
-        return new FixerDefinition(
-            'Single line comments should use double slashes `//` and not hash `#`.',
-            [new CodeSample('<?php # comment')]
-        );
+        return $this->realFixer->isCandidate($tokens);
     }
 
     /**
      * {@inheritdoc}
      */
-    public function isCandidate(Tokens $tokens)
+    public function isRisky()
+    {
+        return $this->realFixer->isRisky();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        return $this->realFixer->fix($file, $tokens);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
     {
-        return $tokens->isTokenKindFound(T_COMMENT);
+        return Utils::formatFixerName(get_called_class());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        return $this->realFixer->getPriority() - 1;
```

Comment:
actually, i don't see much value in defining different priority.
if there is any, add integration test to show it